contracts/GuestList.vy

```diff
@@ -177,31 +175,52 @@ def enough_yfi(user: address, threshold: uint256) -> bool:
 
 # EXTERNAL FUNCTIONS
 
+@view
+@external
+def min_bag() -> uint256:
+    return MIN_BAG
+
+
+@view
+@external
+def ape_out() -> uint256:
+    return APE_OUT
+
+
 @external
 def set_guest(guest: address, invited: bool):
     """
     Invite of kick guests from the party.
     """
-    assert msg.sender == self.bouncer
+    assert msg.sender == self.bouncer  # dev: unauthorized
     self.guests[guest] = invited
 
 
+@external
+def set_bribe_cost(new_cost: uint256):
+    """
+    Set bribe cost denominated in YFI.
+    """
+    assert msg.sender == self.bouncer  # dev: unauthorized
+    self.bribe_cost = new_cost
+
+
 @external
 def set_bouncer(new_bouncer: address):
     """
     Replace bouncer role.
     """
-    assert msg.sender == self.bouncer
+    assert msg.sender == self.bouncer  # dev: unauthorized
     self.bouncer = new_bouncer
 
 
 @external
 def bribe_the_bouncer(guest: address = msg.sender):
     """
-    Sneak into the party by bribing the bouncer with 2% of the entrance cost.
+    Sneak into the party by bribing the bouncer. The pass works for all future vaults.
```

Comment:
Would say "this works for any vault that uses this guest list"

As we may change in the future